Dockerfile

```diff
@@ -39,6 +39,7 @@ COPY docker/start.sh /usr/local/bin/
 COPY docker/start-notebook.sh /usr/local/bin/
 COPY docker/start-singleuser.sh /usr/local/bin/
 COPY docker/jupyter_notebook_config.py /root/.jupyter/
+COPY docker/.bashrc /root/.bashrc
```

Comment:
Nope, it's just nice to have some simple bash aliases.

Virtually no Docker these containers ship with any bash customization (some don't even ship with a shell). I really just got sick of not being able to type `ll`. 😄